working-groups/.template/README.md

```diff
@@ -0,0 +1,100 @@
+#### *<p align="center">This page is an example template and not the actual NLL working group.</p>*
+
+# Non-Lexical Lifetimes (NLL) Working Group
+![working group status: active][status]
+
+This working group aims to implement non-lexical lifetimes (NLL), as described in RFC 2094:
+
+> Extend Rust's borrow system to support non-lexical lifetimes -- these are lifetimes that are
+> based on the control-flow graph, rather than lexical scopes. The RFC describes in detail how
+> to infer these new, more flexible regions, and also describes how to adjust our error messages.
+> The RFC also describes a few other extensions to the borrow checker, the total effect of which is
+> to eliminate many common cases where small, function-local code modifications would be required
+> to pass the borrow check.
+
+**Leads:** [@nikomatsakis][nikomatsakis] and [@pnkfelix][pnkfelix]
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+
+## Goals
+This working group aims to accomplish the following:
+
+- Implement a new borrow checker based on the MIR that supports non-lexical lifetimes.
+- Handle the migration from the existing AST-based borrow checker.
+
+# How can I get involved?
+If you are interested in getting involved in this working group should feel free to assign
+themselves an issue using `@triagebot assign` (look for `E-mentor` and `E-easy` issues
+if you're just getting started) and pop into the Zulip stream to ask for help and advice. If you
+want to get notified on GitHub and on Zulip about NLL issues, then comment with
+`@triagebot subscribe wg-nll` in `#ISSUE_NUMBER`.
```

Comment:
Couple of things... the triagebot doesn't actually work now, right? Also, there are still things like the Zulip user group, which we don't have a story for besides being manually added. Maybe just ask folks to join us at a meeting and/or contact on Zulip?